### DISCUSSION TITLE
Filament Color Change

### DISCUSSION THREAD

**Discussion by PocoComunicativo** [supermerill/SuperSlicer#1112] [Q&A]
I really like Super Slicer. I looked at every configuration, but I couldn´t find the "+" sign at the layer bar (I don't know if that is the name of the bar, but you understand). I think that is very helpful, and I want to know if I can activate that. Thanks...
![11111](https://user-images.githubusercontent.com/19583119/116177031-1ed41500-a6e1-11eb-98a1-28f11eeb38cf.jpg)
I looked for info everywhere, but I didn't found anything

**Comment by neophyl** [comment]
There are several options that disable that, even in Prusa Slicer.  You don't have Sequential Printing enabled do you ?  As that will disable it.

**Reply by PocoComunicativo** [reply]
I had the original configuration, but I just solved it. I printed a temperature tower and I was prompted to save the changes in the profile when I close the program. I selected yes and now I have the plus sign, bjt I can't remember what parameter was changed. Thanks
